src/BackgroundMeshes.jl

```diff
@@ -225,15 +227,12 @@ julia> extrema(x_clip) # should be close to (-1, 1)
 (-0.9912911763341569, 1.0077955853440512)
 ```
 """
-function sigma_clip(
-    x::AbstractArray{T},
-    sigma_low::Real,
-    sigma_high::Real=sigma_low;
-    fill=:clamp,
-    center=median(x),
-    std=std(x),
-) where {T}
-    return sigma_clip!(float(x), sigma_low, sigma_high; fill=fill, center=center, std=std)
+function sigma_clip(x::AbstractArray{T}, sigma_low::Real, sigma_high::Real=sigma_low;
+    fill = :clamp,
+    center = median(x),
+    std = std(x),
+    ) where {T}
+    return sigma_clip!(float(copy(x)), sigma_low, sigma_high; fill, center, std)
```

Comment:
```suggestion
function sigma_clip(x::AbstractArray, sigma_low::Real, sigma_high::Real=sigma_low;
    fill = :clamp,
    center = median(x),
    std = std(x),
    )
    return sigma_clip!(float(copy(x)), sigma_low, sigma_high; fill, center, std)
```
`T` not used in this function